docs/topics_5/web-openshift-deploy.adoc

```diff
@@ -1,28 +1,26 @@
 // Module included in the following assemblies:
 // * docs/web-console-guide_5/master.adoc
 [id='deploy_mta_app_openshift_{context}']
-= Deploying the {ProductShortName} Application
+= Deploying the {ProductShortName} application
 
-. Download the {WebName} from the link:https://developers.redhat.com/products/mta/download[{ProductShortName} Download page].
+. Download the {WebName} from the link:https://developers.redhat.com/products/mta/download[{ProductShortName} download page].
 . Extract the ZIP file to a directory of your choice.
 +
-The path to the directory containing the extracted {ProductShortName} files is referred to as `<MTA_HOME>` throughout this guide.
+The path to this directory is referred to as `_MTA_HOME_` throughout this guide.
 . Access the OpenShift web console. It should appear similar to the following image.
 +
 image::openshift-console.png[]
 
-. Click the `Import YAML / JSON` button in the top-right corner of the web console.
-. Provide the following information in the window that appears, as seen below.
-+
-image::openshift-import-template.png[]
-.. Define a `Project Name`, such as `mta`.
-.. Optionally, define a `Project Display Name` to help describe the project.
-.. Optionally, define a `Project Description` to help provide context on how the project will be used.
-.. Click the **Browse** button and import the desired {WebName} template. These are included in `<MTA_HOME>/openshift/templates`. Once imported, the JSON is visible in the bottom text area.
-. Click the **Create** button to proceed to the next screen.
-. If you would like to save the template as a resource for future projects, check the `Save template` box.
-. Click the **Continue** button to proceed to the next screen.
-. Review the default values provided, adjusting as necessary. For instance, the `web-template-empty-dir-executor.json` file defines 2 CPUs, 4GB of memory, and a 20GB persistent volume.
-. Click the **Create** button to import the template into your project.
-+
+. Click *Import YAML / JSON`* in the upper-right corner of the web console.
+. Fill in the following fields:
+* *Project Name*. Mandatory.
```

Comment:
Put a line break between ordered and unordered lists. It's safer that way. If you don't put in line breaks, occasionally the rendering gets weird.

Reply:
Done. I noticed something that concerned me in the list of files changed. I exchanged the diagrams and screen captures by the same method: using *Save As* for each item. This ensured that each image reference in the files would work -- and the document built fine. But in the list of files, you see that some, but not all, images were recorded as deleted. Is this a problem?

Reply:
I don't think you understand the change that I requested. Or I'm not seeing it.

I also don't understand your question